ui.tests/test-module/specs/formcontainer.spec.js

```diff
@@ -82,6 +82,24 @@ describe('Page/Form Authoring', function () {
             cy.get("[name='./restEndpointPostUrl'").invoke('attr', 'value').should('eq', 'http://localhost:4502/some/endpoint');
         };
 
+    const checkDataModelInEditDialog = function(formContainerEditPathSelector) {
+        // click configure action on adaptive form container component
+        cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + formContainerEditPathSelector);
+        cy.invokeEditableAction("[data-action='CONFIGURE']"); // this line is causing frame busting which is causing cypress to fail
+
+        //open submission tab
+        cy.get('.cmp-adaptiveform-container'+'__editdialog').contains('Data Model').click({force:true});
+        cy.get("[name='./schemaType']").should("exist");
+        cy.get("[name='./schemaRef']").invoke('attr', 'type').should('eq', 'hidden');
+
+        //select email submit action
+        cy.get(".cmp-adaptiveform-container__selectformmodel").children('._coral-Dropdown-trigger').click();
+        cy.get("._coral-Menu-itemLabel").contains('None').should('exist');
+        cy.get("._coral-Menu-itemLabel").contains('Schema').should('be.visible').click();
```

Comment:
pls avoid using internal coral classes (the ones starting with underscore) in tests

Reply:
okay